source/api_documentation/sms_messaging_api/index.rst

```diff
@@ -0,0 +1,144 @@
+    - *N/A* (assumed to be the first page)
+    - *N/A*
+    - False
+
+
+------------------------------------
+Sorting
+------------------------------------
+The records returned by `GET` requests can be sorted using the `sort` URL parameter.
+The value of the `sort` parameter is comprised of:
+
+- an optional valid sortable field
+- an optional sort direction (`asc` or `desc`) separated from the sortable field by a colon (`:`)
+
+For example: `https://example.com?sort=created_at:asc`
+
+If only a sort direction is provided, the response body will be sorted in the provided direction according to the default sortable field.
+
+If only a sortable field is provided, the default sort direction (`asc`) will be assumed.
+
+Each `GET` request denotes the valid sortable fields in its response body. Please see the API examples below for more information.
+
+At this time, responses may be sorted by *one field only*.
+
+**By default, responses are sorted in ascending order according to their created_at field.**
+
+------------------------------------
+Filtering
+------------------------------------
+The records returned by `GET` requests can be filtered using URL parameters.
+Filters are comprised of:
+
+- a valid filterable field
+- a bracketed operator appended to the filterable field
+- a filter value separated from the filterable field and its associated operator by an equal sign (`=`)
+
+For example: `https://example.com?created_at[lte]=2022-01-31` returns all records with a `created_at` value less than or equal to 2022-01-31.
+
+The following operators are supported:
+
+Supported Filter Operators
+------------------------------------
+.. list-table::
+  :widths: 5 50 30
+  :header-rows: 1
+  :class: parameters
+
+  * - Operator
+    - Description
+    - Example
+
+  * - eq
+    - Return only those records where the filtered field is equal to the value found on the right-hand side of the = in this parameter
+    - direction[eq]=in
+
+  * - lte
+    - Return only those records where the filtered field is less than or equal to the value found on the right-hand side of the = in this parameter
+    - created_at[lte]=2022-01-31
+
+  * - gte
+    - Return only those records where the filtered field is greater than or equal to the value found on the right-hand side of the = in this parameter
+    - created_at[gte]=2022-01-01
+
+  * - empty
+    - Return only those records where the filtered field has an empty value such as integer 0, MySQL NULL/null, or an empty string
+    - deleted_at[empty]
+
+
+Each `GET` request denotes the valid filterable fields in its response body. Please see the API examples below for more information.
+
+If a filterable field is defined with a value but does not otherwise include a bracketed operator,
+it is assumed that the `[eq]` operator was intended.
+For example, `direction=incoming` means the same as `direction[eq]=incoming`.
```

Comment:
Same as another comment in this PR, @jbuehring, per [this thread's discussion](https://invoca.slack.com/archives/C042LLXBSHF/p1666908097415999?thread_ts=1666905760.149919&cid=C042LLXBSHF) (and pending further discussion), we may want to remove the above.

Reply:
👑